service/defaultcomponents/defaults.go

```diff
@@ -85,6 +86,7 @@ func Components() (
 	exporters, err := component.MakeExporterFactoryMap(
 		opencensusexporter.NewFactory(),
 		prometheusexporter.NewFactory(),
+		prometheusremotewriteexporter.NewFactory(),
```

Comment:
It is intentional because not yet implemented :)

Reply:
Oh, got it, now I see, sorry for the confusion :)